test/e2e/daemon_restart.go

```diff
@@ -183,8 +183,7 @@ func getContainerRestarts(c *client.Client, ns string, labelSelector labels.Sele
 	return failedContainers, containerRestartNodes.List()
 }
 
-// Flaky issues #17829, #19023
-var _ = Describe("DaemonRestart [Disruptive] [Flaky]", func() {
+var _ = Describe("DaemonRestart [Disruptive]", func() {
```

Comment:
We should be tolerant, but it defeats the purpose of soak; if we're trying to see what happens when clusters are alive for a long time, bonking them often doesn't help us get there.